### ISSUE TITLE
resolution of host.internal variable

### ISSUE THREAD

**Issue by blakeduffey** [sakai135/wsl-vpnkit#236]
When I try to leverage host.internal (which should resolve to the IP of the Windows host) - I get an error.

Does this still exist in 0.4.1?

**Comment by dfillingham** [comment]
No, wsl-vpnkit started using upstream gvproxy so now it's `host.containers.internal`
